CMS-Enterprise/mint-app
# [MINT-2519] [FE] Notification: Data exchange approach is complete

# MINT-2519

## Description


* add `isDataExchangeApproach` typeguard for notification settings
* Update Notification Settings component to take in new Data Exchange Approach notification subscriptions
* Update `GetNotificationSttings` query and its type
* Update i18n file


## How to test this change

1.  Navigate to Notification Settings page and verify it matches figma
2. Subscribe to both notifications types
3. Navigate back to collaboration area and "complete" the Data Exchange form (just fast forward to last page and mark it as complete)
4. Ensure notification shows up on notification page and matches figma
5. Try the unsubscribe link and ensure it works as epected
`http://localhost:3005/notifications/settings?unsubscribe_email=DATA_EXCHANGE_APPROACH_MARKED_COMPLETE`

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
